### PR TITLE
Notebook 6: interactive map viewer with pyodide

### PR DESCRIPTION
**Pull request by zonca**
This goes with https://github.com/cheapandfair/cheapandfair-template/pull/2

### CODE REVIEW

File: 6_extend_catalog.ipynb [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{
```
Comment (rpwagner) [inline 6_extend_catalog.ipynb:1]: <p>Could also <code>cd</code> using the absolute path? <code>/home/jovyan/work/cheapandfair-gateways-2024/heapandfair-template</code>  </p><p>That would also handle the odd case when someone has <code>cd</code>ed  to the work directory.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheapandfair/cheapandfair-gateways-2024/pull/6/discussion/#comment-1778055437'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='1'/>

Comment (rpwagner) [inline 6_extend_catalog.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    url
</pre><p>Will this print out the URL for the student to see it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheapandfair/cheapandfair-gateways-2024/pull/6/discussion/#comment-1778055438'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

**Comment by rpwagner** [comment]
Looks good! The review comments are minor.

File: 6_extend_catalog.ipynb (hunk 1 above)
Comment (zonca) [reply 6_extend_catalog.ipynb:1]: <p>yes</p>

Comment (zonca) [reply 6_extend_catalog.ipynb:1]: <p>I don't like being so tied to the tutorial configuraiton.</p><p>For example that won't work when I am developing on my laptop.</p>

Comment (zonca) [reply 6_extend_catalog.ipynb:1]: <p>the main purpose of this is to avoid issues if someone reruns the cell without restarting the notebook</p>

Comment (rpwagner) [reply 6_extend_catalog.ipynb:1]: Then let's ignore this. You're right, it's likely they're either in one of two directories.

**Pull request merged**